sources/assets/Stride.Core.Assets/PackageSessionPublicHelper.cs

```diff
@@ -41,13 +41,19 @@ public static void FindAndSetMSBuildVersion()
                     ? x.DiscoveryType == DiscoveryType.DotNetSdk && x.Version.Major >= 3
                     : (x.DiscoveryType == DiscoveryType.VisualStudioSetup || x.DiscoveryType == DiscoveryType.DeveloperConsole) && x.Version.Major >= 16);
 
+                if (MSBuildInstance == null)
+                {
+                    throw new InvalidOperationException("Could not find a MSBuild installation (expected 16.0 or later) " +
+                        "Please ensure you have the .NET 6 SDK installed from Microsofts Website");
```

Comment:
Nitpick: "Microsoft's" (with apostrophe) and "website" shouldn't be capitalized. Ideally a link to https://dotnet.microsoft.com/ should be used instead.

Reply:
should be updated now.